lib/presentation/screens/auth/login/login_screen.dart

```diff
@@ -19,12 +19,79 @@ class _LoginScreenState extends State<LoginScreen> {
         listener: (context, state) => _onStateChanged(state),
       );
 
-  // TODO: Implement content
+  final ButtonStyle createAccountButtonStyles = ElevatedButton.styleFrom(
+    primary: Colors.white,
+    onPrimary: Colors.black,
+    minimumSize: const Size(500, 50),
+    shape: const RoundedRectangleBorder(
+        borderRadius: BorderRadius.all(Radius.circular(50)),
+        side: BorderSide(width: 1, color: Colors.white)),
+  );
+
+  final ButtonStyle signInButtonStyles = OutlinedButton.styleFrom(
+    primary: Colors.white,
+    minimumSize: const Size(500, 50),
+    side: const BorderSide(
+      width: 1.0,
+      color: Colors.white,
+    ),
+    shape: const RoundedRectangleBorder(
+      borderRadius: BorderRadius.all(Radius.circular(50)),
+    ),
+  );
+
   Widget _body(LoginScreenState state) => GestureDetector(
         onTap: _onLoginButtonClicked,
         child: Scaffold(
-          backgroundColor: Colors.yellow,
-          body: Center(child: Text(context.localizations.loginText)),
+          backgroundColor: const Color.fromRGBO(255, 87, 100, 1),
+          body: Column(
+            children: [
+              Container(
+                margin: const EdgeInsets.only(top: 20),
+                alignment: Alignment.center,
+                child: Row(
```

Comment:
ok so all those smaller widgets should be split into smaller functions eg.
_logoRow() -> this will contain this row
_policyText() -> policy text and so on
of with this cascade approach few comments on top